django/views/generic/dates.py

```diff
@@ -1,6 +1,7 @@
 import datetime
 
 from django.conf import settings
+from django.contrib.admin.utils import get_fields_from_path
```

Comment:
@timgraham, thank you for the comment. What do you propose? Reimplementing the `get_fields_from_path` inside the `dates.py` for example? Another idea is to extend the Meta options class API?